client/js/store/modules/auth.js

```diff
@@ -107,10 +109,16 @@ export default {
         );
       });
     },
-    async requestStatus(context) {
-      const { location } = window;
-      const isLobby = location.hash.startsWith('#/lobby');
-      const isGame = location.hash.startsWith('#/game');
+    async requestStatus(context, { router, route }) {
+      const { state } = context;
+
+      if (!router || !route || state.authenticated !== null) {
+        return;
+      }
+
+      const { name } = route;
+      const isLobby = name === ROUTE_NAME_LOBBY;
+      const isGame = name === ROUTE_NAME_GAME;
```

Comment:
We can reference the route names by const now, which should dry this up a bit and prevent errors.